setup.py

```diff
@@ -0,0 +1,25 @@
+from distutils.core import setup
+from setuptools import find_packages
+
+dependencies = [
+    'argparse==1.2.1',
+    'funcsigs==0.4',
+    'mock==1.3.0',
+    'pbr==1.8.1',
+    'requests==2.8.1',
+    'six==1.10.0',
+    'wsgiref==0.1.2',
+]
+
+setup(
+    name='pymesync',
+    version='0.0.1',
+    install_requires=dependencies,
+    author=u'OSU Open Source Lab',
+    author_email='support@osuosl.org',
+    packages=find_packages(),
+    url='https://github.com/osuosl/pymesync',
+    license='Apache Version 2.0',
+    description="pymesync - python module for TimeSync API",
+    long_description=open('README.md').read()
```

Comment:
If we want to put this on pypi this line might break. Had bad experiences with referencing static files in setup.py files. Just a warning. I do not have a solution (beyond hardcoding the description in the setup file).